presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -38,12 +38,20 @@ Constructors
 
     Returns a geometry type object from WKT representation.
 
+.. function:: ST_GeometryFromBinary(varbinary) -> Geometry
+
+    Returns a geometry type object from WKB representation.
+
 .. function:: ST_AsText(Geometry) -> varchar
 
     Returns the WKT representation of the geometry. For empty geometries,
     ``ST_AsText(ST_LineFromText('LINESTRING EMPTY'))`` will produce ``'MULTILINESTRING EMPTY'``
     and ``ST_AsText(ST_Polygon('POLYGON EMPTY'))`` will produce ``'MULTIPOLYGON EMPTY'``.
 
+.. function:: ST_AsBinary(Geometry) -> varbinary
+
+    Returns the WKB representation of the geometry.
```

Comment:
Would you specify what does this function return for an empty geometry?

Reply:
@mbasmanova For empty geometries,
    ``ST_AsBinary(ST_LineFromText('LINESTRING EMPTY'))`` will produce ``[01 02 00 00 00 00 00 00   00]``
    and ``ST_AsBinary(ST_Polygon('POLYGON EMPTY'))`` will produce ``[01 03 00 00 00 00 00 00   00]``.  Is this what we want in the docs?

Reply:
@mpertierra I see. Somehow I was thinking that it might produce `null`. Looks like it will actually preserve the type of the geometry which is nice. No, in this case, I think it is fine to skip this detail.